go.mod

```diff
@@ -9,6 +9,7 @@ require (
 	github.com/onsi/ginkgo v1.16.4
 	github.com/onsi/gomega v1.15.0
 	github.com/openshift/api v0.0.0-20211019100638-b2cbe79f2e4b
+	github.com/openshift/cloud-credential-operator v0.0.0-20211115195328-4c537fbd48e9
```

Comment:
Do we still need this dependency knowing that we're getting the root credentials now?